src/shared/util.js

```diff
@@ -919,7 +928,11 @@ function stringToPDFString(str) {
       try {
         const decoder = new TextDecoder(encoding, { fatal: true });
         const buffer = stringToBytes(str);
-        return decoder.decode(buffer);
+        const decoded = decoder.decode(buffer);
+        if (!decoded.includes("\x1B")) {
```

Comment:
For overall consistency, how about using lower-case here as well?
```suggestion
        if (!decoded.includes("\x1b")) {
```